readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -228,27 +225,26 @@ def __init__(self, *args, **kwargs):
 
     def move(self, **__):
         super(HtmlBuilder, self).move()
-        if self.project.has_feature(Feature.BUILD_JSON_ARTIFACTS_WITH_HTML):
-            # Copy json artifacts to its own directory
-            # to keep compatibility with the older builder.
-            json_path = os.path.abspath(
-                os.path.join(self.old_artifact_path, '..', 'json')
+        # Copy JSON artifacts to its own directory
+        # to keep compatibility with the older builder.
```

Comment:
This comment confuses me a little.

What is what we need to keep compatibility to?